backend/config/database.js

```diff
@@ -0,0 +1,29 @@
+const Sequelize = require("sequelize");
+const Secrets = require("./secrets");
+const sequelize =  new Sequelize(
+  Secrets["database"]["url"],
+
+  {
+
+    logging: false,
+    // operatorsAliases: false,
+
+    pool: {
+      max: 15,
+      min: 5,
+      idle: 20000,
+      evict: 15000,
+      acquire: 30000
```

Comment:
These pool parameters seem a little aggressive? I don't see why we'd need to maintain 5 minimum connections to the database - standard use case would probably be one at a time, right?  The defaults are min:0 and max:5. 

And what was the reasoning for adjusting default timeouts? I'm not familiar with best practices, but the defaults according to [documentation](https://sequelize.org/master/class/lib/sequelize.js~Sequelize.html#instance-constructor-constructor) are idle: 10000, evict 1000: acquire: 60000.

Reply:
removed this and using defaults